src/System.IO/src/System.IO.csproj

```diff
@@ -81,9 +81,13 @@
     <TargetingPackReference Include="System.Private.CoreLib.WinRTInterop" />
     <TargetingPackReference Include="System.Private.Interop" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uap101aot'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard13aot'">
     <TargetingPackReference Include="System.Private.Interop" />
+    <ValidateIgnoreReference Include="@(TargetingPackReference)" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'net462' and '$(TargetGroup)' != 'net463' and '$(TargetGroup)' != 'netcore50aot' and '$(TargetGroup)' != 'netstandard13aot'">
```

Comment:
For this project in particular I think I would just remove the TargetPackReference for System.Private.CoreLib which will eliminate the need for this crazy long condition here.